### PR TITLE
RESOPS142 - Remove docker port forwarding

### LINKED ISSUE #142: Remove Docker port forwarding
## Feature Description

Remove (comment) all open ports in Docker Compose except for the proxy service.

blocked by: #12

## Problem You're Trying to Solve

We don't want developers connecting directly to the services, we want them to access everything through the proxy. So let's just not expose any of the service ports.

### PR DESCRIPTION
**Pull request by TobySMurray**
# Description

Commenting out the ports for each service so they interact through the proxy rather than directly.

Closes #142

## How to Test

Services should connect to their data stores without error.

## Checklist
- [ ] The code includes tests if relevant
- [ ] I have *actually* self-reviewed my changes and done QA

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ services:
     depends_on: []
     volumes: [ ./apps/proxy:/app ]
     environment: []
-    ports: [ 80:80, 443:443 ]
+    #ports: [ 80:80, 443:443 ]
```
Comment (MNThomson) [inline docker-compose.yml:21]: The proxy is the only service we can't remove the port forwarding on as this provides the primary entry point to the stack

Comment (TobySMurray) [inline docker-compose.yml:21]:
```suggestion
    ports: [ 80:80, 443:443 ]
```

File: docker-compose.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ services:
     depends_on: []
     volumes: [ ./apps/proxy:/app ]
     environment: []
-    ports: [ 80:80, 443:443 ]
+  ports: [ 80:80, 443:443 ]
```
Comment (TobySMurray) [inline docker-compose.yml:21]:
```suggestion
      ports: [ 80:80, 443:443 ]
```

File: docker-compose.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@ services:
     depends_on: []
     volumes: [ ./apps/proxy:/app ]
     environment: []
-    ports: [ 80:80, 443:443 ]
+      ports: [ 80:80, 443:443 ]
```
Comment (TobySMurray) [inline docker-compose.yml:21]:
```suggestion
         ports: [ 80:80, 443:443 ]
```

Comment (TobySMurray) [inline docker-compose.yml:21]:
```suggestion
      ports: [ 80:80, 443:443 ]
```

Comment (TobySMurray) [inline docker-compose.yml:21]:
```suggestion
    ports: [ 80:80, 443:443 ]
```

**Review by MNThomson — APPROVED** [review]

**Pull request merged**